Autocomplete lt

lauralyeinc opened the pull request:
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

lauralyeinc commented:
I'm still working on this feature, the MAIN feature. I have everything wrote out. Understand it, but have errors in the terminal. My type bars aren't showing up either! Once mock data is showing I'll set it up for review. Thanks everyone!

doanmade commented:
Seeing a few things. lets hop on a zoom call tomorrow and we can go over them. I downloaded this branch to see if i can workout some of the kinks and we can pair program on it tomorrow if you would like.

lauralyeinc commented:
I fixed the merge conflicts. Some of them were in the yarn.lock file, I updated my yarn to see if it was one of the issues with my many tries to set up autocomplete.  App.js is a mess, but if hopefully in the next day or two someone can figure out the cors issues to the API call. Thereafter I will clean up code that is going to work or I can save locally....
